pkg/operator/controllers/node/node_controller.go

```diff
@@ -32,23 +32,24 @@ const (
 	gracePeriod              = time.Hour
 )
 
-// NodeReconciler spots nodes that look like they're stuck upgrading.  When this
+// Reconciler spots nodes that look like they're stuck upgrading.  When this
 // happens, it tries to drain them disabling eviction (so PDBs don't count).
-type NodeReconciler struct {
-	log           *logrus.Entry
+type Reconciler struct {
```

Comment:
Unless there is ambiguity (multiple reconcilers in same package), we just use the name `Reconciler`.